git-multimail/git_multimail.py

```diff
@@ -1503,10 +1503,9 @@ def generate_revision_change_summary(self, push):
                 new_commits_list = []
             new_commits = CommitSet(new_commits_list)
 
+            discarded_commits = CommitSet([])
```

Comment:
I agree that this is a false positive, even in Python3. I don't think we should worsen up the code to deal with a broken tool.

Reply:
I agree too. The lgtm people seem rather reactive so reporting the false positive may actually fix the issue (on the lgtm side).